cdc/model/http_model.go

```diff
@@ -31,6 +31,16 @@ func (t JSONTime) MarshalJSON() ([]byte, error) {
 	return []byte(stamp), nil
 }
 
+// UnmarshalJSON used to parse time.Time from bytes
+func (t *JSONTime) UnmarshalJSON(data []byte) error {
+	tm, err := time.Parse(`"2006-01-02 15:04:05.000"`, string(data))
```

Comment:
```suggestion
	tm, err := time.Parse("2006-01-02 15:04:05.000", string(data))
```

typo?

Reply:
The time layout can be string literal, see https://go.dev/ref/spec.
This line can also be written as:
`tm, err := time.Parse("\"2006-01-02 15:04:05.000\"", string(data))`